### DISCUSSION TITLE
Does bg-opacity-{x} still work on v3.0?

### DISCUSSION THREAD

**Discussion by zoltanszogyenyi** [tailwindlabs/tailwindcss#6444] [Help] [answered]
Hey everyone,

Congrats on the v3.0 launch!

One question: do the `bg-opacity-{x}` classes still work? I've noticed you improved the syntax but looks like some background opacities are breaking after the update for my project.

If it's a breaking change I think it should be mentioned somewhere.

Thanks!

**Comment by bodograumann** [comment]
Seems to work where I use it, but given that it was removed from the docs, is it deprecated @adamwathan ?
Is it not possible to change background opacity independantly from background color in the future?

**Comment by adamwathan** [comment, marked as answer]
Definitely still works and will for the foreseeable future, but considered deprecated in favor of the opacity modifier.
